### DISCUSSION TITLE
how can I use sprites?

### DISCUSSION THREAD

**Discussion by Salam797** [moononournation/Arduino_GFX#424] [Q&A]
I am using an esp32 qualia dev board and TL040HDS20-B1502A Square RGB 666 TTL TFT Display - 4" 720x720. I want to create sprites to generate smooth animations and eliminate flicker but I don't know where to get started. please help

**Comment by moononournation** [comment]
You may start with SpriteGif.ino

**Reply by Salam797** [reply]
> You may start with SpriteGif.ino

Thanks for your response sir. 
It's a bit complicated for me as I'm a beginner. Could you please share how I can initiate the code using the following display: Square RGB TTL TFT Display - 4" 720x720 - with Capacitive Touch - TL040HDS20CT-B1502A, using adafruit ESP32 Qualia?

In addition, I would like to know what sprite library I should use for arduino gfx?. For example for (TFT eSPI) library , TFT_eSprite is used.

Please advise.

You are assistance is greatly appreciated.

**Reply by Salam797** [reply]
> You may start with SpriteGif.ino

Sir please can you make a step by step video explaining how to create a sprite using arduino gfx library?
 
I really need help with this as the sprite gif code compiles and uploads but my screen stays blank. Your assistance is greatly appreciated.
